src/performance/perf-update.test.ts

```diff
@@ -0,0 +1,59 @@
+import { check} from "k6";
+import http from "k6/http";
+import *  as type from "k6/index";
+import { getConfigOrThrow } from "../utils/config";
+
+const config = getConfigOrThrow();
+
+export let options = {
+    scenarios: {
+        contacts: {
+            executor: "constant-arrival-rate",
```

Comment:
Instead of running an executor  **constant-arrival-rate** why don't we use an executor **ramping-vus**